addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -27,18 +27,18 @@
                         <span invisible="not project_id">(<field name="progress" class="oe_inline" nolabel="1" widget="integer"/> %)</span>
                     </div>
                 </xpath>
+                <xpath expr="//header" position="after">
+                    <div class="alert alert-warning text-center mb-2" role="alert" invisible="analytic_account_active" groups="hr_timesheet.group_hr_timesheet_user">
+                        You cannot log timesheets on this project since it is linked to an inactive analytic account. <br/> Please change this account, or reactivate the current one to timesheet on the project.
```

Comment:
```suggestion
                        You cannot log timesheets on this project since it is linked to an inactive analytic account.<br/> Please change this account, or reactivate the current one to timesheet on the project.
```